cronjobs/indexcontent.php

```diff
@@ -62,22 +62,24 @@
                     if ( !( $node instanceof eZContentObjectTreeNode ) )
                     {
                         $cli->error( "An error occured while trying fetching node $nodeId" );
+                        $db->query( "DELETE FROM ezpending_actions WHERE action = '$action' AND param = '$objectID'" );
+                        $db->commit();
```

Comment:
Won't this db `commit()` potentially introduce a transaction counter mismatch somewhere?